src/account.ts

```diff
@@ -106,6 +106,21 @@ export abstract class Account {
         return this.starknetContract.address;
     }
 
+    async getAccountBalance(): Promise<StringMap> {
+        const abiPath = process.cwd() + "/contract-artifacts/ERC20/ERC20_abi.json";
+
+        const hre = await import("hardhat");
+        const ethContract = new StarknetContract({
+            abiPath: abiPath,
+            hre: hre
+        });
+        ethContract.address = "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
```

Comment:
This could be a constant in `src/constants.ts`

Reply:
Refactored accordingly 👍